homeassistant/components/openhome/config_flow.py

```diff
@@ -24,6 +24,9 @@ def _is_complete_discovery(discovery_info: SsdpServiceInfo) -> bool:
 class OpenhomeConfigFlow(ConfigFlow, domain=DOMAIN):
     """Handle an Openhome config flow."""
 
+    host: str | None
+    name: str
```

Comment:
Would it make sense to make these private? (and encourage it in other contributions)
```suggestion
    _host: str | None
    _name: str
```

Reply:
Note: You write "make these private", your suggested code however, you show them protected and not private...

Reply:
Thanks for the reminder 👍

Reply:
Yeah, it's a good point.